src/middleware/retrieveTCPTransaction.js

```diff
@@ -7,6 +7,19 @@ export async function koaMiddleware (ctx, next) {
 
   ctx.body = transaction.data
   ctx.authorisedChannel = transaction.channel
+  ctx.isTcpChannel = true
```

Comment:
@bradsawadye im not sure why this property is added to the context? was this missing?

Reply:
@BMartinos it was not missing. This is a property that is used in the streamingReceiver middleware. The streaming middleware is purely for http requests. There are cases when we are routing from TCP to Http, and in these cases, we use the streamingReceiver middleware. This property together with 'tcpChannelHasHttpRoute' property are used to determine whether to bypass the streamingReciever or not